base_layer/wallet/src/text_message_service/handle.rs

```diff
@@ -55,13 +58,30 @@ pub enum TextMessageResponse {
     ContactUpdated,
 }
 
-pub struct TextMessageServiceRequester {
-    handle: TextMessageHandle,
+/// Events that can be published on the Text Message Service Event Stream
+#[derive(Debug, Hash, PartialEq, Eq)]
+pub enum TextMessageEvent {
+    ReceivedTextMessage,
```

Comment:
Could we perhaps add the text message to the event? I'd imagine something consuming this event stream would need the data (or some ID to get the data) along with the event.